enclave_manager/setup.py

```diff
@@ -77,8 +77,9 @@
     os.path.join(enclave_bridge_wrapper_path, 'work_order_wrap.cpp'),
     os.path.join(enclave_bridge_wrapper_path, 'enclave_info.cpp'),
     os.path.join(enclave_bridge_wrapper_path, 'signup_info.cpp'),
-    os.path.join(enclave_bridge_wrapper_path, 'signup_info_kme.cpp'),
-    os.path.join(enclave_bridge_wrapper_path, 'signup_info_wpe.cpp'),
+    os.path.join(enclave_bridge_wrapper_path, 'signup_info_singleton.cpp'),
+#    os.path.join(enclave_bridge_wrapper_path, 'signup_info_kme.cpp'),
+#    os.path.join(enclave_bridge_wrapper_path, 'signup_info_wpe.cpp'),
```

Comment:
Why is this commented ? Is it due to any build issues ?

Reply:
This was commented because i had changes for other workers, but didn't include it here.
I will remove the commented code.